### PR TITLE
Conan repository description

### PR DESCRIPTION
**Pull request by ChGen**
For #1 - Here's initial information on Conan repository internals & structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+Conan is a C/C++ repository, this adapter is an SDK for working with Conan data and metadata and a HTTP endpoint for the Conan repository.
+
+
+## How does Conan work
+
+* Repository data with Conan recipes is the directory with fixed structure. *.yml is used for metadata, and *.py is used for packaging logic. Binary artifacts stored in separate repository. https://docs.conan.io/en/latest/creating_packages/getting_started.html
+* Basic configuration options: OS (Windows/Linux/macOS), architecture, compiler, static/shared mode, debug/release mode. https://docs.conan.io/en/latest/creating_packages/define_abi_compatibility.html
+* Conan uses Python scripts with some base classes provided by Conan project, which wrap steps like patch/configure/make/install, as well as dependencies configuration.
+* Makefile/Automake and CMake are supported well by Conan python framework, custom builds could also be implemented manually. https://docs.conan.io/en/latest/reference/build_helpers.html
+* When accepting repository changes it builds for multiple popular configurations before code review
```
Comment (g4s8) [inline README.md:12]: @ChGen where the actual build happens? Could you please describe a workflow of working with repository? If I understand correctly it is (please fix me if I'm wrong):
 1. Client prepare build recipes (Makefiles, Conan `.py` scripts, CMake scripts)
 2. Client builds binaries and upload to dedicated? server (not same as Conan repository server?)
 3. Client uploads package metadata to Conan repository server
 4. Server regenerates repository metadata by merging new package

File: README.md [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
+* After reviewing and merging binary artifacts are stored for the most popular configurations. Other configurations built locally on client
+* Every artifact recipe contains simple executable test project, which is also used for build & packaging checks
```
Comment (g4s8) [inline README.md:14]: @ChGen Does Conan repo perform some tests and builds?

File: README.md [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
+* Conan provides its own server on Python. SSL/https and authentication is supported but not required. https://docs.conan.io/en/latest/uploading_packages/running_your_server.html
```
Comment (g4s8) [inline README.md:15]: @ChGen it looks like a proxy for remote Conan repository

File: README.md [hunk 4]
```diff
@@ -1,17 +1,28 @@
...
+* Conan supports multiple remotes for multiple sources, including company-internal hosting. https://docs.conan.io/en/latest/uploading_packages/remotes.html
+* Binary artifacts repository for every artifact stores archived source code and multiple pre-built packages with its built configuration in specified text format (conaninfo.txt)
```
Comment (g4s8) [inline README.md:17]: @ChGen if Conan repo and binary repo are different server, does Conan repo know about location of binary repo?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@ChGen please check my comments and questions

File: README.md [hunk 5]
```diff
@@ -1,17 +1,28 @@
...
-
+# Conan Artipie adapter
+
+Conan is a C/C++ repository, this adapter is an SDK for working with Conan data and metadata and a HTTP endpoint for the Conan repository.
+
+
+## How does Conan work
+
+* Repository data with Conan recipes is the directory with fixed structure. *.yml is used for metadata, and *.py is used for packaging logic. Binary artifacts stored in separate repository. https://docs.conan.io/en/latest/creating_packages/getting_started.html
+* Basic configuration options: OS (Windows/Linux/macOS), architecture, compiler, static/shared mode, debug/release mode. https://docs.conan.io/en/latest/creating_packages/define_abi_compatibility.html
```
[the remark this answers is not in the document]
Comment (ChGen) [reply README.md:9]: Its about structure of the Conan recipe (artifact) configuration(s). I will clarify it in the next commit.

File: README.md [hunk 6]
```diff
@@ -1,17 +1,28 @@
...
+* Conan uses Python scripts with some base classes provided by Conan project, which wrap steps like patch/configure/make/install, as well as dependencies configuration.
```
[the remark this answers is not in the document]
Comment (ChGen) [reply README.md:10]: They are executed server side (when checking recipe or preparing binary artifacts) and client side, when preparing package or when binary package is missing for client's configuration.

File: README.md [hunk 7]
```diff
@@ -1,17 +1,28 @@
...
+* Makefile/Automake and CMake are supported well by Conan python framework, custom builds could also be implemented manually. https://docs.conan.io/en/latest/reference/build_helpers.html
```
[the remark this answers is not in the document]
Comment (ChGen) [reply README.md:11]: It is related to conan recipe (artifact). Every artifact can have multiple configurations.

File: README.md (hunk 1 above)
Comment (ChGen) [reply README.md:12]: I'll describe it in the next commit

File: README.md (hunk 2 above)
Comment (ChGen) [reply README.md:14]: Yes, Conan repo (in wider term) performs test builts & test run before manual reviewing & merging recipes.

File: README.md (hunk 3 above)
Comment (ChGen) [reply README.md:15]: In a way, yes.

File: README.md (hunk 4 above)
Comment (ChGen) [reply README.md:17]: I think their test servers know about both servers & repos, since they perform different checks, builds & synchronization.

**Review by g4s8 — APPROVED** [review]
@ChGen thanks, looks good

**Pull request merged**